build.gradle.kts

```diff
@@ -1,20 +1,21 @@
-import java.util.Calendar
 import nl.javadude.gradle.plugins.license.LicensePlugin
 import org.checkerframework.gradle.plugin.CheckerFrameworkPlugin
+import java.util.*
 
 plugins {
```

Comment:
Could clean a lot up by using Indra.

Reply:
I prefer not to use it; haven't touched it before, and the build file is clear in what it does without knowing how Indra works as-is.